NGitLab.Mock/Repository.cs

```diff
@@ -67,9 +68,11 @@ private LibGit2Sharp.Repository GetGitRepository()
                     if (_directory == null)
                     {
                         var directory = TemporaryDirectory.Create();
+                        var repositoryDirectory = Path.Combine(directory.FullPath, Project.PathWithNamespace.Replace('/', Path.DirectorySeparatorChar));
```

Comment:
Can you add a comment to explain why having the namespace in the path is important